pkg/config/config.go

```diff
@@ -120,6 +121,10 @@ func (c *Config) fillDefaults() error {
 		AdmissionPolicy: RouteAdmissionPolicy{
 			NamespaceOwnership: NamespaceOwnershipAllowed,
 		},
+		Ports: IngressPortsConfig{
+			Http:  &(&struct{ x int }{80}).x,
```

Comment:
Is this just to get a pointer to the integer? We already have a dependency on https://pkg.go.dev/k8s.io/utils/pointer and that might be easier to read.

Reply:
Turns out that package was deprecated and had to use k8s.io/utils/ptr which uses generics. Let me know what you think.